functions.php

```diff
@@ -153,7 +156,13 @@ function enqueue_styles() : void {
  * @return void
  */
 function enqueue_editor_styles() : void {
-	wp_enqueue_style( 'theme-editor-tweaks', get_stylesheet_directory_uri() . '/assets/editor-tweaks.css', [ 'theme-style-variables' ], wp_get_theme()->get( 'Version' ) );
+	wp_enqueue_style(
+		'theme-editor-tweaks',
+		MANY_ASSETS_URL . '/editor-tweaks.css',
```

Comment:
Could this be added to the main `add_editor_style` function call above?

Reply:
No, as that function runs stuff through the prepending system in Gutenberg. This CSS file actually tweaks Gutenberg itself, to distinguish the background colors.

That said, `theme-blocks-styles` should be included, but I'm not sure that `add_editor_style` allows references to enqueued scripts.